lib/ex_quiz_it/questions/enums.ex

```diff
@@ -0,0 +1,38 @@
+defmodule ExQuizIt.Questions.Enums do
+  @moduledoc false
+  def q_and_a do
+    {:docs_v1, _, :elixir, _, _, _, functions} = Code.fetch_docs(Enum)
+
+    functions
+    |> Stream.map(&doc_parser(&1))
+    |> Stream.map(&get_answer(&1))
```

Comment:
You can use the `&function/1` syntax here

Reply:
Yep ....